deepmd/dpmodel/fitting/general_fitting.py

```diff
@@ -166,6 +169,9 @@ def __init__(
                     self.resnet_dt,
                     self.precision,
                     bias_out=True,
+                    seed=seed + ii * (len(self.neuron) + 1)
```

Comment:
shall we define a class method telling the number of random seeds used by one fitting net?

Reply:
Now it accepts a `list[int]`, so this is not needed anymore.